dnf-behave-tests/features/steps/shell.py

```diff
@@ -27,8 +27,12 @@ def when_I_open_dnf_shell(context):
     # pexpect adds a short delay before sending data, so that SSH has time
     # to turn off TTY echo; the echo is still there though, so removing the delay
     context.shell_session.delaybeforesend = None
-    
-    context.shell_session.expect('> ')
+
+    try:
+        context.shell_session.expect('> ')
+    except pexpect.exceptions.EOF:
+        stdout_from_shell(context)
+        raise AssertionError("dnf shell unexpected output")
```

Comment:
I think a better error text would be "dnf shell terminated unexpectedly" or something along those lines?